content/100-getting-started/02-setup-prisma/100-start-from-scratch/110-relational-databases/200-install-prisma-client.mdx

```diff
@@ -19,7 +19,7 @@ Notice that the install command automatically invokes `prisma generate` for you
 
 ![Install and generate Prisma Client](../../../images/prisma-client-install-and-generate.png)
 
-Whenever you update your Prisma schema, you will have to update your database schema using either `prisma migrate dev` or `prisma db push`. This will keep your database schema in sync with your Prisma schema. The commands will also regenerate Prisma Client to update your Prisma Client API.
+Whenever you update your Prisma schema, you will have to update your database schema using either `prisma migrate dev` or `prisma db push`. This will keep your database schema in sync with your Prisma schema. The commands will also regenerate Prisma Client.
```

Comment:
The information that this updates the Prisma Client API to match the Prisma schema is relevant, I don't think we can assume people understand that intuitively.